README.md

```diff
@@ -0,0 +1,345 @@
+---
+permalink: /guides/guide-codewind/
+layout: guide-markdown
+title: Getting Started with Codewind and Kabanero
+duration: 30 minutes 
+description: Take advantage of Codewind's tools to help build high quality cloud native applications regardless of which IDE or language you use.
+tags: ['kabanero', 'appsody', 'codewind', 'microservice']
+projectid: guide-codewind
+page-layout: guide-markdown
+page-duration: 30 minutes 
+page-releasedate: 02-21-2020
+page-guide-category: stacks
+page-essential: true
+page-essential-order: 1
+page-description: Take advantage of Codewind's tools to help build high quality cloud native applications regardless of which IDE or language you use. 
+guide-author: Kabanero
+page-tags: ['kabanero', 'appsody', 'codewind', 'microservice']
+page-permalink: /guides/{projectid}
+repo-description: TBD
+---
+
+# Getting Started with Codewind and Kabanero
+
+## Objectives
+
+### Learning objectives 
+
+In this guide you will learn how to develop a simple microservice with your chosen IDE, using Eclipse Codewind, an open source project, which provides IDE extensions for popular IDEs (VS Code and Eclipse IDE).  Eclipse Codewind provides the ability to create new projects based on application stack hub that include application stacks customized to meet your company policies and consistently deploy applications and microservices at scale.
+
+## Overview 
+
+Kabanero's developer experience for IDEs is provided by Eclipse Codewind.  For more information on how Kabanero and its components work, visit [Kabanero's Architecture and Development Workflows](https://kabanero.io/docs/ref/general/overview/architecture-overview.html).
+
+Kabanero installs on OpenShift and integrates a modern DevOps toolchain and application stack hubs which enable developers to use runtimes and frameworks in pre-built container images called `Application stacks`.  
+
+Eclipse Codewind provides the ability to create application projects for these `Application Stacks` that your company has built, enabling developers to focus on their code and not infrastructure and Kubernetes.  Application deployments to Kubernetes occur via pipelines when developers commit their local code to the correct Git repos Kabanero is managing via webhooks.    
+
+Eclipse Codewind provides the ability to create projects based on a variety of different template types.  These include IBM Cloud starters,Odo, and Appsody templates.  When used with Kabanero, you will only use Appsody templates for the Collections your company has enabled.  Today, there are collections for: IBM Cloud Starters, OpenShift Do (odo), Eclipse MicroProfile/Java EE, Springboot, Node.js, Node.js with Express, Node.js with Loopback.
```

Comment:
Still mentions of Collections in this paragraph. Infact, now that Collections have gone, the "When used with Kabanero .." sentence doesn't make much sense. Perhaps remove this sentence altogether and replace "collections" in the sentence that follows with something else. Do you mean "stacks" or "templates"?  Templates makes more sense I think.